posthog/year_in_posthog/2023.html

```diff
@@ -462,7 +462,7 @@ <h1 class="highlight text-3xl">{{ human_badge }}</h1>
         </div>
         {% if badges %}
             <div class="achievements-wrapper pt-2">
-                <h2 class="highlight text-xl">Achievements unlocked {{ achievements_count }}/7</h2>
+                <h2 class="highlight text-xl">Achievements unlocked {{ achievements_count }}/8</h2>
```

Comment:
Should we make this a variable so that we don't forget to change it next year when we launch 6 more new products?